transform/mattermost-analytics/dbt_project.yml

```diff
@@ -48,6 +48,9 @@ models:
       web_app:
         schema: int_web_app
         tags: ['nightly']
+      hightouch:
```

Comment:
The name must be the name of the target team, not of the target integration. I think in this case it should be `sales`.

Reply:
Updated this to `int_sales`